src/test/resources/resources/elastic-resource.json

```diff
@@ -0,0 +1,6 @@
+{
+  "@id": "http://resources.nexus.com/test-resource/306e8f68-8419-11e8-adc0-fa7ae01bbebc",
+  "project": "http://admin.nexus.example.com/projects/bbp/testproject",
```

Comment:
Where this project is coming from? so far only [this metadata](https://github.com/wwajerowicz/nexus-kg/blob/685e3cfa2d068abb7d4346008c1dca595e225d84/src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/ResourceF.scala#L74) is stored on a ElasticSearch document

Reply:
I missed that, I need to add it to the indexer.

Reply:
I would say the project uuid is needed as a metadata, but not the project URI (because it might change, since the labels can be updated).

In any case, for your logic, you should not need the project URI anywhere, since in the routes you already have the project label and the account label